style.css

```diff
@@ -0,0 +1,3 @@
+body{
```

Comment:
kindly apply the appropriate line spacing in your css code to make the code much more readable and easy to pass through